src/Tools/dotnet-monitor/DiagnosticPortValidateOptions.cs

```diff
@@ -12,10 +14,29 @@ internal class DiagnosticPortValidateOptions :
     {
         public ValidateOptionsResult Validate(string name, DiagnosticPortOptions options)
         {
+            var failures = new List<string>();
+
             if (options.ConnectionMode == DiagnosticPortConnectionMode.Listen
                 && string.IsNullOrEmpty(options.EndpointName))
             {
-                return ValidateOptionsResult.Fail(Strings.ErrorMessage_DiagnosticPortMissingInListenMode);
+                failures.Add(Strings.ErrorMessage_DiagnosticPortMissingInListenMode);
+            }
+
+            // On Windows, the server is implemented using Named Pipes.
+            // -1 means use as many as resources allow.
+            // Otherwise, the allowed instances are between 1 and 254.
```

Comment:
```suggestion
            // Otherwise, the allowed instances are between 1 and 254, inclusive.
```

Reply:
I'm going to keep this as is, since I copied this message from the NamedPipeServerStream exception.